.yarnrc

```diff
@@ -0,0 +1,6 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't think we want to add this file (and also anything under `.yarn`) to the repo, you should probably add `.yarn` and `.yarnrc` to `.gitignore`